Documentation/upgrading.md

```diff
@@ -1,173 +1,143 @@
-    kube-apiserver-vyg3t                       2/2       Running   0          1h
-    kube-controller-manager-1709527928-zj8c4   1/1       Running   0          47m
-    kube-dns-v20-3531996453-0tlv9              3/3       Running   0          1h
-    kube-proxy-6dbne                           1/1       Running   0          1s
-    kube-proxy-sm4jv                           1/1       Running   0          8s
-    kube-proxy-xmuao                           1/1       Running   0          14s
-    kube-scheduler-2255275287-hti6w            1/1       Running   0          49m
-    kubelet-hfdwr                              1/1       Running   0          38s
-    kubelet-oia47                              1/1       Running   0          52s
-    kubelet-s6dab                              1/1       Running   0          59s
-    pod-checkpointer-node1.example.com         1/1       Running   0          1h
-
-## Verify
-
-Verify that the kubelet and kube-proxy on each node have been upgraded.
+### Verify
 
-    $ kubectl get nodes -o yaml | grep 'kubeletVersion\|kubeProxyVersion'
-          kubeProxyVersion: v1.4.3+coreos.0
-          kubeletVersion: v1.4.3+coreos.0
-          kubeProxyVersion: v1.4.3+coreos.0
-          kubeletVersion: v1.4.3+coreos.0
-          kubeProxyVersion: v1.4.3+coreos.0
-          kubeletVersion: v1.4.3+coreos.0
+Verify the control plane components updated.
 
-Now, all self-hosted Kubernetes components have been upgraded to a new version of Kubernetes!
+```sh
+$ kubectl version
+Client Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.2", GitCommit:"477efc3cbe6a7effca06bd1452fa356e2201e1ee", GitTreeState:"clean", BuildDate:"2017-04-19T20:33:11Z", GoVersion:"go1.7.5", Compiler:"gc", Platform:"linux/amd64"}
+Server Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.7+coreos.0", GitCommit:"c8c505ee26ac3ab4d1dff506c46bc5538bc66733", GitTreeState:"clean", BuildDate:"2017-07-06T17:38:33Z", GoVersion:"go1.7.6", Compiler:"gc", Platform:"linux/amd64"}
+```
 
-## On-host Kubelet
+```sh
+$ kubectl get nodes
+NAME                               STATUS    AGE       VERSION
+node1.example.com                  Ready     21d       v1.6.7+coreos.0
+node2.example.com                  Ready     21d       v1.6.7+coreos.0
+node3.example.com                  Ready     21d       v1.6.7+coreos.0
+node4.example.com                  Ready     21d       v1.6.7+coreos.0
+```
 
-The on-host kubelet is started by systemd, and is used to launch, and be replaced by, the self-hosted kubelet. While the on-host kubelet is short-lived, it should be kept up to date with the self hosted components.
+## kubelet
 
-On each host running a kubelet, modify the kubelet systemd unit to reference a new `KUBELET_IMAGE_TAG` (an example snippet is below)
+SSH to each node and update `/etc/kubernetes/kubelet.env`. Restart the `kubelet.service`.
 
-`/etc/systemd/system/kubelet.service`
+```sh
+ssh core@node1.example.com
+sudo vim /etc/kubernetes/kubelet.env
```

Comment:
This doesn't exist in our hack examples. Instead we directly set the env vars in the service: https://github.com/kubernetes-incubator/bootkube/blob/master/hack/quickstart/kubelet.master#L2-L3

Reply:
SSH to each node and update the `KUBELET_IMAGE_TAG` in `kubelet.service` or `/etc/kubernetes/kubelet.env`, depending on the provisioning tool used. Restart the `kubelet.service`.